src/Symfony/Component/Messenger/composer.json

```diff
@@ -18,6 +18,7 @@
     "require": {
         "php": ">=7.2.5",
         "psr/log": "^1|^2|^3",
+        "psr/event-dispatcher": "^1",
```

Comment:
This is technically only required if using the queue parts of the library, I think.  If it's purely a sync message bus it's not required.  That suggests it may be better as a suggests/dev.

PSR packages are small enough that it doesn't really matter, but it's worth mentioning as event-dispatcher wasn't in a first-level require.  I'm comfortable either way.

Reply:
It was, because we could have Symfony's EventDispatcher contracts v1 installed which extended the PSR-14 interface conditionally (= only if it's present), mainly because of PHP 7.1 compatibility (PSR-14 requires PHP 7.2).

I have pushed a new version that should work even in that combination, which allowed me to remove the dependency again.